watson.zsh-completion

```diff
@@ -0,0 +1,186 @@
+#compdef watson
+
+
+# ~~~ LIST OF COMMANDS ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+_watson_commands=(
+  'cancel:Cancel the last call to the start command.'
+  'config:Get and set configuration options.'
+  'edit:Edit a frame.'
+  'frames:Display the list of all frame IDs.'
+  'help:Display help information'
+  'log:Display each recorded session during the...'
+  'merge:Perform a merge of the existing frames with a...'
+  'projects:Display the list of all the existing...'
+  'remove:Remove a frame.'
+  'report:Display a report of the time spent on each...'
+  'restart:Restart monitoring time for a previously...'
+  'start:Start monitoring time for the given project.'
+  'status:Display when the current project was started...'
+  'stop:Stop monitoring time for the current project.'
+  'sync:Get the frames from the server and push the...'
+  'tags:Display the list of all the tags.'
+)
+
+
+# ~~~ LOCATING THE FRAMES FILE ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+if [[ -n "$WATSON_DIR" ]]; then
+  _watson_frame_file=${WATSON_DIR}/frames
+elif [[ "$OSTYPE" =~ "^darwin" ]]; then
+  _watson_frame_file=${HOME}/Library/Application Support/frames
+else
+  _watson_frame_file=${HOME}/.config/watson/frames
+fi
```

Comment:
I really think this is the way to go. An alternative would be to get the output of `EDITOR=echo watson config -e` but that a ugly hack imo.